### PR TITLE
Reworked plotting script

### PR DESCRIPTION
**Pull request by NikodemKch**
Includes all requested types of plots except "iterations vs. difference" plot. Also, there is a small TODO which however should not have impact on the end result.

### CODE REVIEW

**Comment by vGsteiger** [comment]
How many runs did you do per plot? I would assume around 5-10 runs per flag/complexity and so forth? Then I would introduce plots with whiskers to have the std deviation and so forth.

**Comment by NikodemKch** [comment]
The main script now has a batch mode which creates the CSV needed for the plots. Added whiskers to plots.

**Comment by vGsteiger** [comment]
Could you remove the log, the csvs, the init c files and add the new parameter to the readme? Other than that perfect!

File: plotting/data_complexity.csv
Comment (vGsteiger) [inline plotting/data_complexity.csv:1]: Please remove

File: plotting/data_optimizations.csv
Comment (vGsteiger) [inline plotting/data_optimizations.csv:1]: Remove

File: plotting/data_single.csv
Comment (vGsteiger) [inline plotting/data_single.csv:1]: Remove

File: plotting/data_timeouts.csv
Comment (vGsteiger) [inline plotting/data_timeouts.csv:1]: Remove

File: plotting/srcreduce_cleaned.log.keep
Comment (vGsteiger) [inline plotting/srcreduce_cleaned.log.keep:1]: Remove

File: testing_input/init0.c
Comment (vGsteiger) [inline testing_input/init0.c:1]: Remove

File: src/srcreduce/main.py [hunk 1]
```diff
@@ -146,14 +188,25 @@ def new_run(args):
     logging.info("The best code was %s", best_code_path)
     logging.info("with heuristic %f", best_code_heuristic)
     logging.info("derived from %s", best_code_init)
-    # Used to print info in the end
-    calculate_heuristic_value(args, best_code_init, best_code_path)
+
+    # Used to print info in the end and retrieve info in batch mode
+    info_dict = {}
+    if args.batch_measurements is not None and not save_iters:
```
Comment (vGsteiger) [inline src/srcreduce/main.py:194]: Please add to readme :)

**Review by vGsteiger — CHANGES_REQUESTED** [review]
As written, thanks for the work!

**Comment by NikodemKch** [comment]
Everything fixed, Percentile class improved.

**Review by vGsteiger — APPROVED** [review]
Great thank you!

**Pull request merged**